.env.dev

```diff
@@ -0,0 +1,2 @@
+MAX_PARALLELISM=5000
+MAX_BATCHED_JOBS_ITERATIONS=2
```

Comment:
at this point both values are the same as the defaults so it could be removed - I used this to test it with multiple values to find the best settings.